custom_components/tuya_local/devices/orison_chanfok_neo_fan_light.yaml

```diff
@@ -0,0 +1,102 @@
+name: Orison Chanfok Neo
+products:
+  - id: eba992532f93bb1c009bjw
```

Comment:
This looks like a device id rather than a product id.  Is the product id perhaps `zfwgeelmqcypcdgh` as per the previous comment?